src/GraphQL/GraphQL.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="GraphQL-Parser" Version="5.3.3" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
+    <PackageReference Include="System.Memory" Version="4.5.4" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
I am absolutely sure that we will need this package in the near future. In addition, if we target the package also to netstandard2.1, then all the necessary APIs will be available out of the box, no code changes required.

Reply:
~ +800 KB dependencies but it's worth it

Reply:
I just have a hard time thinking that this change is worth another dependency in the final version of v3 before we move to v4.  Isn't this code only used when building a schema?

For v4, I'd be happy to multi-target .net standard 2.0 and 2.1, adding system.buffers or similar MS-provided dependencies if we need to.

Reply:
> Isn't this code only used when building a schema?

I see a lot of optimizations even without multi-targeting.